### PR TITLE
[mono] Add Vector128 Abs intrinsics for Amd64

### PR DESCRIPTION
**Pull request by matouskozak**
Add support for the following Vector128 API's:

- Abs

### CODE REVIEW

**Review by vargaz — APPROVED** [review]

File: src/mono/mono/mini/simd-intrinsics.c [hunk 1]
```diff
@@ -1126,12 +1126,28 @@ emit_sri_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsi
...
+
+		if (type_enum_is_float(arg0_type)) {
+			// args [0] & ~vector(-0.0)
+			MonoInst *zero = emit_xzero(cfg, arg_class);	// 0.0
+			zero = emit_simd_ins (cfg, klass, OP_NEGATION, zero->dreg, -1); // -0.0
+			MonoInst *ins = emit_simd_ins (cfg, klass, OP_SSE_ANDN, zero->dreg, args [0]->dreg);
+			ins->inst_c1 = arg0_type;
+			return ins;
+		} else {
+			// Works correctly even when SSSE3 is not supported or when using I8
```
Comment (fanyang-mono) [inline src/mono/mono/mini/simd-intrinsics.c:1148]: We shouldn't need this comment to explain what the following line of code is doing. Need to replace `OP_SSSE3_ABS` with a better name.

Comment (matouskozak) [reply src/mono/mono/mini/simd-intrinsics.c]: Replaced with `OP_VECTOR_IABS`.

**Review by fanyang-mono — APPROVED** [review]

**Pull request merged**